docs/sources/reference/commandline/cli.md

```diff
@@ -1434,12 +1434,14 @@ Running `docker ps` showing 2 linked containers.
 
     $ sudo docker ps
     CONTAINER ID        IMAGE                        COMMAND                CREATED              STATUS              PORTS               NAMES
-    4c01db0b339c        ubuntu:12.04                 bash                   17 seconds ago       Up 16 seconds                           webapp
+    4c01db0b339c        ubuntu:12.04                 bash                   17 seconds ago       Up 16 seconds       3300-3310/tcp       webapp
     d7886598dbe2        crosbymichael/redis:latest   /redis-server --dir    33 minutes ago       Up 33 minutes       6379/tcp            redis,webapp/db
 
 `docker ps` will show only running containers by default. To see all containers:
 `docker ps -a`
 
+`docker ps` will group exposed ports into a single range if possible, e.g. a container that exposes TCP ports `100, 101, 102` will display `100-102/tcp` in the `PORTS` column.
```

Comment:
Run-on. Need to break this into two sentences starting at E.g. Need a comma after the "g." as well, please!